src/Pages/OfficerDB/Edits/Adder.js

```diff
@@ -0,0 +1,215 @@
+import React, { useState } from 'react'
+import {
+  Modal,
+  ModalHeader,
+  ModalBody,
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+  Input
+} from 'reactstrap'
+import ConfirmModule from './ConfirmationModule'
+import Uploader, { upload, getFileName, checkUpload } from './Uploader'
+
+const enums = require('../../../Enums')
+const api = require('../../../APIFunctions/OfficerDB')
+
+export default function Adder (props) {
+  const [personDropdown, setPersonDropdown] = useState(false)
+  const [roleDropdown, setRoleDropdown] = useState(false)
+  const [person, setPerson] = useState('')
+  const [role, setRole] = useState('')
+  const [linkedIn, setLinkedIn] = useState('')
+  const [github, setGithub] = useState('')
+  const [quote, setQuote] = useState('')
+  const [queryUsers, setQueryUsers] = useState('')
+  const optionalInputs = [
+    {
+      placeholder: 'linkedIn',
+      onChange: event => setLinkedIn(event.target.value)
+    },
+    {
+      placeholder: 'Github',
+      onChange: event => setGithub(event.target.value)
+    },
+    {
+      placeholder: 'Quote',
+      onChange: event => setQuote(event.target.value)
+    }
+  ]
+
+  function findUser (query) {
+    query.toLowerCase().trim()
+      ? setQueryUsers(
+        props.users.filter(user =>
+          user.firstName.toLowerCase().includes(query)
+        )
+      )
+      : setQueryUsers('')
+  }
+
+  async function submitOfficer () {
+    let accessLevel = ''
+    if (enums.getAllKeys(enums.officerRole).includes(role)) {
```

Comment:
The additional functions in the enums file is getting confusing. By using numbers or constant strings, we don't have to worry about these things.

Sure we would store the role as 1, 2, or 3 but in the code they are reflected as constant values, e.g. DEVELOPMENT, EVENT_PLANNING, etc.

Can we change the enums to reflect this? See [here](https://github.com/SCE-Development/Core-v4/blob/b4707f9f5ef75f3806b7cc6d2cce5f81454e90f6/src/Enums.js#L6)

Reply:
actually, the values indicate the access-level of each role.
If i use an object then I wouldn't be able to do ```js map ``` but I need it for the drop-downs

Reply:
Why not make one `Role` array, ranging from banned to executive?

Reply:
We could. I wanted to keep them separated from one another in case we need to use the chair array or the officer array separately in another component; so you dont get a bunch of irrelevant role when you only wanted chairs or officers